src/processes/filters.rs

```diff
@@ -59,6 +63,18 @@ impl QueryFilter {
         s.to_lowercase().contains(&self.query)
     }
 
+    fn query_match_fuzzy_str(&self, s: &str) -> bool {
+        let score = self.matcher.fuzzy_match(s, self.query.as_str());
+        // TODO: fine-tune the score threshold or make it configurable?
```

Comment:
I think if this unrevealed bigger problem.
As far as I understand fuzzy search:
Query "a" -> all processes that contains 'a' in cmd should be displayed but all of them will have low score right?
Current solution would return these processes despite the fact that we matched only single char

Reply:
Yeah, but on the other hand, do we really want to show results when giving only 1 or 2 letters? I mean, how relevant are they in such a case?

Reply:
Most of the time they are not, but think of a command that is one or 2 chars long like fd or rg
Besides i think this is how killing with fzf actually works.

Reply:
But you are adding fuzzy match for path
`SearchBy::Path => self.query_matches_fuzzy_opt(prc.cmd_path()),`
Which means `/if/bin/path/is/long/xx` ten score may be low for `xx` query right?
I don't have perfect answer to this problem I'm only hightlighting some potential issues someone may face.
I'm not sure though how score would work in above situation and if someone might be confused why pik didn't show `xx` command
I wouldn't mind if pik shown me all results after query just '/a' as this is live search and I got what I wanted -> paths containing letter 'a'

Reply:
I'll add some tests to see how it works in such scenarios, but I understand what you mean.

Reply:
> I think if this unrevealed bigger problem. As far as I understand fuzzy search: Query "a" -> all processes that contains 'a' in cmd should be displayed but all of them will have low score right? Current solution would return these processes despite the fact that we matched only single char

Right, but I discovered we can just put `s >= 0` to make it work exactly as the current solution. We might want to increase the score if we need to decrease the amount of non-relevant lines. 

> Which means /if/bin/path/is/long/xx ten score may be low for xx query right?

I also checked this. The answer is no. The score will be the same for both `/a` and `/very/long/path/to/a`.

But I think in order to improve the UX, we need to sort the results before showing them to the user. Now, the less relevant rows may pop up first.

Reply:
We may add sort in next PR's
I prefer small changes too see what is going on